test/e2e/config/conf.js

```diff
@@ -6,6 +6,17 @@ chai.use(chaiAsPromised);
 
 const argv = minimist(process.argv.slice(2));
 
+//const specFilesFilter = ['../features/**/*.feature'];
```

Comment:
can u remove this or make a comment why its commented out

Reply:
feature improvements